source/MRViewer/MRUISaveChangesPopup.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <functional>
+
+#include "exports.h"
+
+namespace MR
+{
+
+namespace UI
+{
+
+struct SettingsSaveChangesPopup
+{
+    // menu scaling
+    float scaling = 1.0f;
+    // the text for the button, if you need to exit the application
+    std::string out = "Sign out";
```

Comment:
dont use "Sign Out" as default text
```suggestion
    // text that is shown if we have nothing to save
    std::string shortCloseText = "Close";